script.js

```diff
@@ -0,0 +1,230 @@
+let menu = document.querySelector(".menu")
+let ham = document.querySelector(".ham")
+let xIcon = document.querySelector(".xIcon")
+let menuIcon = document.querySelector(".menuIcon")
+const main =  document.querySelector('.main')
+const header = document.getElementById('logo')
+const projs = document.querySelectorAll('.work')
+
+
+// function projOn() {
+//   projs.forEach(
+//     (work) => {
+//       work.style.filter = 'blur(4px)';
+//     },
+//   );
+// }
+
+// function projOff() {
+//   projs.forEach(
+//     (work) => {
+//       work.style.filter = 'blur(0px)';
+//     },
+//   );
+// }
```

Comment:
Please remove commented code because it makes the whole project unclean and inefficient.